### ISSUE TITLE
Create From Current Editor Theme causes multiple errors

### ISSUE THREAD

**Issue by Listwon** [godotengine/godot#20178]
**Godot version:**
Godot 3.0.5 stable 64-bit

**OS/device including version:**
Windows 10
NVS 3100M on T510 laptop
GTX 1080 on PC

**Issue description:**
Multiple errors, probably triggered when GUI is repainted after creating theme from current editor theme.

In editor:
`ERROR: texture_get_data: Condition ' !texture->active ' is true. returned: Ref<Image>() At: drivers/gles3/rasterizer_storage_gles3.cpp:835`

In runtime:
```
ERROR: texture_set_size_override: Condition ' p_width <= 0 || p_width > 16384 ' is true.
   At: drivers/gles3/rasterizer_storage_gles3.cpp:1040
ERROR: create_from_image: Condition ' p_image.is_null() ' is true.
   At: scene/resources/texture.cpp:191

```
Creating empty theme or removing it fixes errors.

**Steps to reproduce:**
1. Create new project
2. Add any Control type node
3. Create new theme for this node
4. From "Edit theme..." select "Create From Current Editor Theme" and save it
5. Try clicking, moving, losing focus on nodes on the current scene
6. Try running the project

**Comment by squished-01** [comment]
Confirmed and it's annoying.   Appears to be related to generating a preview thumbnail of the scene underneath the scene tabs in the top center.

**Comment by squished-01** [comment]
I've found out how to get rid of the errors.

You want to check all the StyleBoxTextures, and any that give you an error when you click the texture it points to, save the texture as a resource.  And you want to make sure none of the Icons point to a texture that has a null image.  The below exact steps are what worked for me:

Open the tres file for the theme you created from current editor theme
For each control type in HScrollBar, HSplitContainer, ProgressBar, VScrollBar, VSplitContainer:
--> For each StyleBoxTexture in the Styles folder:
-------> click the StyleBoxTexture to open it
-------> click the texture to open that
-------> save the texture as... (anything)
For each control type in HScrollBar, VScrollBar:
---> Clear each Icon, setting them to <null> 

Dont forget to save the theme.

Error spam BEGONE! 

Hopefully someone smarter than me can figure out how to fix the bug in Godot so the above isn't necessary.

**Comment by PLyczkowski** [comment]
@squished-01 I'm unable to replicate your success, I get crashes : ( Could you maybe post the debugged .tres theme file you made? Preferably from the Gryy theme.. I would be very grateful.

**Comment by AlexHolly** [comment]
Maybe related

https://github.com/godotengine/godot/commit/ab9bbce3114d476bc00bf19aa5eaec6998f29514
x11

Selecting a Node in Scene Tree uses 100% CPU

1. Create a new Theme for a Control
2. Use "Create From Current Editor Theme"
3. Create a new Control and select between first and second Node.

**Comment by bojidar-bg** [comment]
@AlexHolly That would be #23163, no?

**Comment by AlexHolly** [comment]
this seems to be fixed by https://github.com/godotengine/godot/commit/8b231b96e347b677ea4189784c960bc4517b8e6a

**Comment by akien-mga** [comment]
Nice, closing then.

**Comment by Listwon** [comment]
I confirm that it's fixed :) (just tested this on current Godot 3.1 beta 10 build)
